src/components/AssetsTable/index.jsx

```diff
@@ -526,6 +544,7 @@ export default class AssetsTable extends React.Component {
         }
       </WrappedMessage>
     );
+
     return (
       <React.Fragment>
```

Comment:
This doesn't need to be a `React.Fragment` too, right?

Reply:
It has to be, since the two WrappedMessage elements are adjacent and need to be wrapped in something.